tfp-automation/Jenkinsfile

```diff
@@ -24,7 +30,7 @@ node {
     stage('Run Module Test') {
             def dockerImage = docker.image('tfp-automation')
             dockerImage.inside() {
-                sh "go test -v -timeout 1h -run ${params.TEST_CASE} ./tfp-automation/tests/cluster"
+                sh "go test -v -timeout ${timeout} -run ${env.TEST_CASE} ./tfp-automation/tests/cluster"
```

Comment:
I think the `params.TEST_CASE` is okay to leave if you want. I am not sure if `env.TEST_CASE` will break or not.